OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -16,17 +16,25 @@ void TracerThread::Run(
   Reset();
 
   // perf_event_open refers to cores as "CPUs".
-  int32_t num_cpus = GetNumCores();
+  std::vector<int32_t> all_cpus;
```

Comment:
I think there should be a comment here that explains why you use "all_cpus" and "cpuset_cpus". All CPUs for context switches etc. and CPUs based on cpuset for dynamic instrumentation (and why this is sufficient and what we want).